www/lendingbot.js

```diff
@@ -370,6 +376,30 @@ function doSave() {
     });
     localStorage.setItem('timespanNames', JSON.stringify(timespanNames))
 
+    // Bitcoin Display Unit
+    localStorage.displayUnitText = $('input[name="btcDisplayUnit"]:checked').val();
```

Comment:
I also agree about setters and getters, however this is already in use in other places in this file. I actually just reused the method.